### ISSUE TITLE
Edit credit for Discussion Teamwork image for redesigned homepage

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3131]
### Overview
We need to edit a credit for a new image used in the redesigned version of the homepage so that the [Credits page](https://www.hackforla.org/credits/) is up to date.

### Details
From the [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724), here is a screenshot of the credit's information for reference:
  <details>
    <summary>Click here to see the screenshot of the Discussion Teamwork credit in Figma</summary>
     <img src="https://user-images.githubusercontent.com/37763229/168177805-ac022586-b4eb-43c2-a17c-898f2330b12b.png" alt="">
  </details>

### Action Items
- [x] **Feature Branch:** You will be working off a feature branch called `feature-homepage-launch` and the pull request created from this issue should be merged into that feature branch. Please read and follow the steps in the [How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch) wiki article.
- [x] For `_data/internal/credits/product-management.yml`,
  - [x] Change the file name from `product-management.yml` to `discussion-teamwork.yml`
  - [x] Change `title: Product Management` to `title: Discussion Teamwork`
  - [x] Change `content: icon` to `content-type: image`
  - [x] Change `used-in: Home` to `used-in: Homepage`
  - [x] Change `alt: 'Product Management Icon'` to `alt: ''`
  - [x] Remove `type: icon`
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3130 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3130 to the New Issue Approval column and remove the Dependency label
  
### Resources/Instructions
- This issue is tracked in the epic #3130.
- [Credits page](https://www.hackforla.org/credits/)
- [How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- [Credits yml file template for Iconography and Imagery](https://github.com/hackforla/website/wiki/Credits-yml-file-template-for-Iconography-and-Imagery)
- [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724)

**Comment by ExperimentsInHonesty** [comment]
This issue would usually have the milestone 01. Compliance.  However, it's specifically for the homepage launch, which has not happened yet.

**Comment by oliviaywangn** [comment]
Availability; 3 hrs
ETA: 1.5 week

**Comment by BeckettOBrien** [comment]
Availability: 3 Hours
ETA: EOD 7/7/22

**Comment by JessicaLucindaCheng** [comment]
Not sure why the automations are not automatically linking the pr to this issue even though it has `Fixes #3131` in https://github.com/hackforla/website/pull/3338#issue-1297824354.

I have gone ahead and manually linked pr #3338 to this issue. See this comment: https://github.com/hackforla/website/pull/3338#issuecomment-1179632289

I will now manually close this issue as completed.
